src/freenet/client/filter/HTMLFilter.java

```diff
@@ -3001,6 +3020,24 @@ Map<String, Object> sanitizeHash(Map<String, Object> h,
 					} else if (name.equalsIgnoreCase("Viewport")) {
 						hn.put("name", name);
 						hn.put("content", content);
+					} else if (name.equalsIgnoreCase("referrer")) {
+						hn.put("name", name);
+						hn.put("content", "no-referrer"); // Only no-referrer is allowed in freesites for privacy
```

Comment:
What privacy exactly? This is about what Referer header is sent upon navigation, but considering the destination would always be the node that rendered this page in the first place (external links are bounced through an intermediate page), there's nothing to hide.

Also the implementation does not match the comment - this does not "only allow" `no-referrer`, but forces `no-referrer` whenever any value is set (but does not enforce it when **no** value is set).

I would suggest dropping support for this metadata name.